src/msg/async/Stack.h

```diff
@@ -245,6 +247,8 @@ class Worker {
     plb.add_u64_counter(l_msgr_send_bytes, "msgr_send_bytes", "Network sent bytes", NULL, 0, unit_t(UNIT_BYTES));
     plb.add_u64_counter(l_msgr_active_connections, "msgr_active_connections", "Active connection number");
     plb.add_u64_counter(l_msgr_created_connections, "msgr_created_connections", "Created connection number");
+    plb.add_u64_counter(l_msgr_connection_timeouts, "msgr_connection_timeouts", "Number of connection timeouts", NULL, PerfCountersBuilder::PRIO_USEFUL);
```

Comment:
```suggestion
    plb.add_u64_counter(l_msgr_connection_ready_timeouts, "msgr_connection_ready_timeouts", "Number of not yet ready connections declared as dead", NULL, PerfCountersBuilder::PRIO_USEFUL);
```

Reply:
@idryomov should be fixed by now

Reply:
No, still missing `msgr` prefix in one case...